src/styles/page-template/index.md.njk

```diff
@@ -241,6 +241,12 @@ To change the components that are included in the page template by default, set
       </td>
     </tr>
 
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">opengraphImageUrl</td>
+      <td class="govuk-table__cell">Variable</td>
+      <td class="govuk-table__cell">Set the URL for the Open Graph image meta tag. Must be an absolute URL, including the protocol and domain name.</td>
```

Comment:
A small thing, but can the second sentence in the description start something like - You must use an absolute URL... or It must be an absolute URL... ?

Reply:
I've changed it to "The URL must be absolute, including the protocol and domain name." – does that work?

Reply:
Yup, looks great.